content/id/docs/concepts/storage/dynamic-provisioning.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Penyediaan Volume Dinamis
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+
+Penyediaan volume dinamis memungkinkan volume penyimpanan untuk dibuat sesuai permintaan (_on-demand_).
+Tanpa adanya penyediaan dinamis (_dynamic provisioning_), untuk membuat volume penyimpanan baru, admin kluster haruslah secara manual
```

Comment:
haruslah secara manual -> secara manual harus

Reply:
addressed in 490d25dd8a13b115293327a3e7ad9009052f2212